san/get.py

```diff
@@ -9,9 +9,20 @@
     'ohlcv': 'get_ohlcv'
 }
 
+DEPRECATED_QUERIES = {
+    'mvrv_ratio': 'mvrv_usd',
+    'nvt_ratio': 'nvt',
+    'realized_value': 'realized_value_usd',
+    'token_circulation': 'circulation',
```

Comment:
the replacement should be `circulation_1d` I think

Reply:
You're right, thanks

Reply:
Is it the same with realized_value and realized_value_usd/_1d ?